oslc4j-jena-provider/src/main/java/org/eclipse/lyo/oslc4j/provider/jena/AbstractOslcRdfXmlProvider.java

```diff
@@ -274,7 +274,9 @@ private String getSerializationLanguage(final MediaType baseMediaType) {
 		mediaPairs.add(new AbstractMap.SimpleEntry<>(OslcMediaType.TEXT_TURTLE_TYPE,
 													 RDFLanguages.strLangTurtle));
 		mediaPairs.add(new AbstractMap.SimpleEntry<>(OslcMediaType.APPLICATION_JSON_LD_TYPE,
-													 RDFLanguages.strLangJSONLD));
+				 RDFLanguages.strLangJSONLD));
+		mediaPairs.add(new AbstractMap.SimpleEntry<>(OslcMediaType.TEXT_XML_TYPE,
```

Comment:
Why did you suggest to remove this?

re:
package org.eclipse.lyo.oslc4j.provider.jena;

@Provider
@Produces({OslcMediaType.APPLICATION_XML, OslcMediaType.TEXT_XML})
@Consumes({OslcMediaType.APPLICATION_XML, OslcMediaType.TEXT_XML})
public class OslcXmlProvider extends OslcRdfXmlProvider {

This provider is already registered, but doesn't work properly because the getSerializationLanguage isn't included in the list.

Reply:
@jamsden That's a change I believe you also made on the 3rd commit of this branch.